src/main/java/jalau/at18/katas/phonenumber/maria/CreatePhoneNumber.java

```diff
@@ -0,0 +1,42 @@
+package jalau.at18.katas.phonenumber.maria;
+
+public class CreatePhoneNumber {
+
+    public static final int THREE = 3;
+    public static final int FOUR = 4;
+    public static final int FIVE = 5;
+    public static final int SIX = 6;
+    public static final int SEVEN = 7;
```

Comment:
You might not need all these constants, if you think about it, if you add a unit test for this code, you could remove main() method and it contents, because with the unit test, you already verify that your code works as expected, so you don't need main() method anymore to do manual verification